core/sequencer/sequencer_test.go

```diff
@@ -13,101 +13,3 @@
 // limitations under the License.
 
 package sequencer
```

Comment:
Should this file be deleted?

Reply:
It's empty isn't it? There should be more tests in here.